docs/content/sites.md

```diff
@@ -0,0 +1,105 @@
+# Sites
+
+Static files can easily be deployed and served from a Wrangler project.
+
+### Setup new project
+
+If you want to setup a new project use  `wrangler generate` . Examples of how to setup a project for Hugo, Gatsby and React can be seen in our [template gallery](TODO).
+
+To set up a generic project you can run:
+
+```
+wrangler generate --site myProj
+```
+
+This command creates the source directory `dist`, `wrangler.toml`, and `worker-site` directory required. 
+
+Any files you wish to upload and make live should live in `dist`. You can change this later.
+
+### Setup existing static project
+
+If you already have a folder with the static files you'd like to be served from a Worker script, you can upload them to Worker's KV. First run:
+
+```
+$ wrangler init --site
+```
+
+This command creates the `wrangler.toml` and `worker-site` directory required to upload the files and serve them from our network. 
+
+### Config Bucket
+
+Special keys in your `wrangler.toml` for using sites with Worker scripts are:
+
+| Key             | Value                                                        | Example                              |
+| --------------- | ------------------------------------------------------------ | ------------------------------------ |
+| `bucket`        | The file path to the static assests you wish to serve in respect to the directory containing your wrangler.toml | `bucket = "./dist"`                  |
+| \*`entry-point` | The location of your Worker script, default is `workers-site` which is autogenerated for you | ```entry-point = "./workers-site"``` |
+
+\* Note you shouldn't need to touch these values.
+
+Add this site `bucket` to your `wrangler.toml`:
+
+```
+name = "my-blog"
+type = "webpack"
+account_id = "43434f8d0d9f52a395a1472b35f23439" 
+site = { bucket = "./public", entry-point = "./workers-site" }
```

Comment:
nit: since wrangler generates this with bracket notation, we may as well be consistent and use that:
```suggestion
[site]
bucket = "./public"
entry-point = "./workers-site"
```